ast/astVisit.ml

```diff
@@ -392,3 +392,24 @@ let collect_accesses_in_position statement { Location.line; column } =
         | _ -> None
     end) in
   Collector.collect (Source.create [statement])
+
+
+let contains_yield {Statement.Define.body; _} =
+  let module YieldVisitor = struct
+    type t = bool
+
+    let expression result expression =
+      match (result, expression) with
+      | (true, _) -> true
+      | (false, {Node.value = Expression.Yield _; _}) -> true
+      | (false, _) -> false
+
+    let statement result statement =
+      match (result, statement) with
+      | (true, _) -> true
+      | (false, {Node.value = Statement.Yield _; _}) -> true
+      | (false, {Node.value = Statement.YieldFrom _; _}) -> true
+      | (false, _) -> false
+  end in
```

Comment:
We care about consistency in the code so here go a few style-nits. As a rule of thumb it's usually a good idea to be consistent with the surrounding code.

- I'd inline the module as above
- `in`s get their own separate line unless the statement fits on a single line (the one in the previous function is also inconsistent with the rest of our code, whoops ;) )
- space after `{` and before `}`
- The parentheses around the tuples are not necessary.

Reply:
Done.